tests/Microsoft.Bot.Builder.Tests/TranscriptUtilities.cs

```diff
@@ -29,14 +29,14 @@ public static class TranscriptUtilities
         /// Loads a list of activities from a transcript file.
         /// Use the context of the test to find the transcript file.
         /// </summary>
-        /// <param name="context">Test context.</param>
+        /// <param name="className">Class name.</param>
+        /// <param name="testName">Test name.</param>
         /// <returns>A list of activities to test.</returns>
-        public static IEnumerable<IActivity> GetFromTestContext(TestContext context)
```

Comment:
>GetFromTestContext(TestContext context) [](start = 45, length = 39)

Since we are not passing TestContext anymore, the method name does not seem to make sense anymore.

Reply:
updated.